WTC_Libs/stateMachine/comms.c

```diff
@@ -147,10 +147,9 @@ void commsToWTC(uint8_t* buffer) {
 }
 
 void commsToPis(uint8_t* buffer, uint8_t pi) {
-	char prompt[100] = { };
-	char commandFromPi[100] = { };
-	char stopPipeCommand[100] = { };
-	sprintf(commandFromPi, "\0");
```

Comment:
This sprintf was unnecessary, since it only prints a null pointer into a string. Initializing the arrays to zero means every byte is already 0.